compose/dev/kepler-dev/etc/kepler/config.yaml

```
@@ -36,6 +36,11 @@ exporter:
     debugCollectors:
       - go
       - process
+    # metricsLevel: all
+    metricsLevel:
```

```suggestion
    metricsLevels:
```
to indicate multiple